migrations/20220725204351_more-project-data.sql

```diff
@@ -0,0 +1,37 @@
+-- Add migration script here
+ALTER TABLE mods_categories
+    ADD COLUMN is_additional BOOLEAN NOT NULL DEFAULT FALSE;
+
+ALTER TABLE mods
+    ADD COLUMN approved timestamptz NULL;
+
+ALTER TABLE categories
+    ADD COLUMN header varchar(256) NOT NULL DEFAULT 'Categories';
```

Comment:
Maybe we could add a `description` field while we're at it?

Reply:
this pr is pretty big in scope, but this is out of scope, it's not really related to the changes needed for resourcepacks/plugins and we don't have descriptions for all the categories at the moment